phockup.py

```diff
@@ -116,27 +134,33 @@ def main():
         default=-1,
         choices=range(0, 255),
         metavar="1-255",
-        help="Descend at most 'maxdepth' levels (a non-negative integer) of directories",
+        help="""\
```

Comment:
I don't get this formatting here. I understand the line wrapping of the multiline comments but this one can be on the same line. Is this a PEP8 requirement?

Reply:
It's not a PEP8 requirement for PEP8 to do it like this as long it's within 79 characters per line. I thought everywhere in the code to keep the same formatting.